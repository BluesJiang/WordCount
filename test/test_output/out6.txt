test\test_cases\test2.in, ×Ö·ûÊı: 168
test\test_cases\test2.in, µ¥´ÊÊı: 15
