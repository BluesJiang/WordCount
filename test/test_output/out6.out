test\test_cases\test2.in, 字符数: 168
test\test_cases\test2.in, 单词数: 15
